# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/xmms-crystality/cflags.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur crystality-plugin-0.92.orig/Makefile crystality-plugin-0.92/Makefile
--- crystality-plugin-0.92.orig/Makefile	2001-08-13 01:41:14.000000000 +0200
+++ crystality-plugin-0.92/Makefile	2004-10-23 19:14:05.000000000 +0200
@@ -7,7 +7,7 @@
 STDIO_PLUGIN=	crystality-stdio
 STDIO_LIBS=	-lpthread $(shell gtk-config --libs)
 
-CFLAGS= -s -mpentium -O6 -Wall -ffast-math -fomit-frame-pointer -finline-functions -fexpensive-optimizations $(GTK_CFLAGS)
+CFLAGS= -s -Wall -ffast-math -fomit-frame-pointer -finline-functions -fexpensive-optimizations $(GTK_CFLAGS)
 
 
 TARGETS=	$(XMMS_PLUGIN) $(STDIO_PLUGIN)
